cfgov/unprocessed/apps/youth-employment-success/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "youth-employment-success",
+  "description": "The consumerfinance.gov/practitioner-resources/resources-youth-employment-programs/transportation-tool/",
+  "homepage": "https://www.consumerfinance.govpractitioner-resources/resources-youth-employment-programs/transportation-tool/",
+  "author": {
+    "name": "Consumer Financial Protection Bureau",
+    "email": "tech@cfpb.gov",
+    "url": "https://cfpb.github.io/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cfpb/cfgov-refresh.git"
+  },
+  "private": true,
+  "license": "SEE LICENSE IN TERMS.md",
+  "dependencies": {
+    "keymirror": "0.1.1"
```

Comment:
Noted earlier… I'd probably refactor to omit keymirror, particularly since it's the only dependency included.